packages/waku/src/lib/plugins/vite-plugin-rsc-hmr.ts

```diff
@@ -33,7 +33,7 @@ export function rscHmrPlugin(): Plugin {
       return [
         {
           tag: 'script',
-          attrs: { type: 'module' },
+          attrs: { type: 'module', async: true },
```

Comment:
By default it's "defer" and the script isn't evaluated during HTML streaming.
cc @Aslemammad

Reply:
niceeee, i love that solution.